Uyanda.Coffee.Integration/AlphaVantage/Integrations/AlphaVantageIntegration.cs

```diff
@@ -0,0 +1,41 @@
+﻿using System;
```

Comment:
Please have this class file one folder up. Just in _AlphaVantage_ instead of in _AlphaVantage/Integration_

Reply:
file moved up one folder